components/ConvertSteps/ConvertSteps.js

```diff
@@ -65,13 +72,16 @@ function ConvertSteps({ toAnj, fromAmount, toAmount, onReturnHome, steps }) {
         // Awaiting confirmation
         const transaction = await createTx()
 
-        onHashCreated && onHashCreated(transaction.hash)
+        hashCreated && hashCreated(transaction.hash)
```

Comment:
For the sake of naming, I wonder if we should be consistent with these APIs and leave the `on` prefix there?